src/Aspire.Dashboard/Components/Controls/GridValue.razor

```diff
@@ -18,9 +18,16 @@
     }
     else
     {
-        <span class="cellText" title="@(ToolTip ?? Value)">
-            @(MaxDisplayLength.HasValue ? TrimLength(Value) : Value)
-        </span>
+        @if (IsMultilineValue is true && Value is not null)
+        {
+            <FluentTextArea Rows="@GetNumberOfLines(Value)" Resize="TextAreaResize.Both" Value="@Value" ReadOnly="true" />
```

Comment:
Could we try something more like a div with `white-space: pre` (and `overflow: auto` or whatever is needed for scrolling, plus probably a max height)? I think that'd probably look cleaner than the readonly text area. You _may_ even be able to get away with not special-casing as much about multi-line in that case (since most values won't be multi-line). But you'd probably have to try it out to see.